ts/features/bonus/bonusVacanze/components/buttons/ButtonConfigurations.ts

```diff
@@ -35,10 +39,12 @@ export const confirmButtonProps = (
  */
 export const errorButtonProps = (
   onPress: () => void,
-  title?: string
+  title?: string,
+  icon?: string
```

Comment:
```suggestion
  iconName?: string
```
To avoid misleading if icon is an image or an image-name